app/controllers/admin_controller.rb

```diff
@@ -17,30 +17,31 @@ class AdminController < ApplicationController
   def switch_locale(&action)
     locale =
       Rails.application.config.country[:dashboard_locale].presence ||
-        http_accept_language.compatible_language_from(I18n.available_locales) ||
-        I18n.default_locale
+      http_accept_language.compatible_language_from(I18n.available_locales) ||
+      I18n.default_locale
 
     I18n.with_locale(locale, &action)
   end
 
   def root
     redirect_to default_root_paths.find { |policy, _path|
-      DashboardPolicy.new(pundit_user, :dashboard).send(policy) }.second
+                  DashboardPolicy.new(pundit_user, :dashboard).send(policy)
```

Comment:
This is weird, but the fact that the underlying code is using `send` like this is actually the bigger warning flag. 😨

Reply:
This indentation is wonky.

The `send` issue, I think it's just some metaprogramming, and we could very well use `public_send` instead. Looks like we're not accessing private methods, just invoking a bunch of Pundit policy actions.